### PR TITLE
Removed upper limit of django-otp dependency

### PR DESCRIPTION
**Pull request by claudep**
django-otp recently released a 1.0.0 version.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     packages=find_packages(exclude=('example', 'tests')),
     install_requires=[
         'Django>=2.2',
-        'django_otp>=0.6.0,<0.99',
+        'django_otp>=0.6.0',
```
Comment (moggers87) [inline setup.py:16]: Do you think it's safe to have no upper limit? django-otp does publish quite often and has been known to break things at random. Will that change post 1.0.0?

Comment (claudep) [reply setup.py:16]: I think that tests should show when some `django_otp` release breaks things. That can happen with any other package, I don't really see a good reason to special-case `django_otp` at this regard.

**Comment by moggers87** [comment]
Oops, that should have been approved, not request for changes.

**Comment by moggers87** [comment]
@claudep ???

**Comment by claudep** [comment]
Sorry, I thought this was merged already! I guess I have to recreate a new PR.

**Comment by moggers87** [comment]
There should be a button to restore the branch.

**Comment by claudep** [comment]
Ah yes, thanks!!

**Comment by moggers87** [comment]
Thanks! :+1:

**Pull request merged**